.vscode/settings.json

```diff
@@ -15,6 +15,9 @@
     "**/node_modules": true,
     "**/*.js": {
       "when": "$(basename).ts"
+    },
+    "**/*.jsx": {
```

Comment:
Ah this is maybe what I was having trouble with when trying to set up the package and import tsx component into test app.

Reply:
I doubt that. This setting is to tell VS code don't show the generated JSX files if you find a file with the same basename but with TSX extension. 
I think your problem might have been that `jsx: react` option in the `tsconfig.json`

Reply:
Ah I see,  I had `"jsx": "react-jsx"`,  and thought I tried the other variations as well,  but maybe some other misconfiguration on my side